### ISSUE TITLE
Support for hiding the default back button 

### ISSUE THREAD

**Issue by zjaml** [react-community/react-navigation#123]
I want to hide back button for some screens in StackNavigator.
So far I've tried to do the following without success. Is it not supported yet or am I doing anything wrong.
The right button does appear though.

(In a individual screen)
```
static navigationOptions = {
    title: "First Filter",
    header: ({ state, setParams, goBack, navigate }) => ({
      left:  <Button title="left"/>
      right: <Button title="right"/>
    })
  }
```
I've also tried to set header.left to null, no luck there also.

Thanks for this great library.

**Comment by grabbou** [comment]
Hey, support for that landed later https://github.com/react-community/react-navigation/commit/7c0fcc6cf58a0a601dc4749801dc68f28d4b0307. Can you check with this commit instead of `npm` version?

**Comment by satya164** [comment]
@grabbou this doesn't check for `null`, so unlikely that it works - `if (header && header.left) {`

**Comment by zjaml** [comment]
@grabbou  Thank you, I confirmed with the master branch. it works when I set header.left as <View/>, but not with null as @satya164 pointed out.

**Comment by grabbou** [comment]
We already added support for that in my PR (pending review), so this will get fixed with #109
